src/compas_viewer/scene/sceneobject.py

```diff
@@ -178,7 +179,7 @@ def __init__(
         self.background: bool = False
 
         #  Geometric
-        self.transformation: Optional[Transformation] = None
+        self.transformation: Optional[Transformation] = kwargs.get("transformation", None)
```

Comment:
I would add a line `transformation=None,` in init instead of saying kwargs.get here